docs/pubsub-api.rst

```diff
@@ -0,0 +1,251 @@
+``gcloud.pubsub`` API
+=====================
+
+Connection / Authorization
+--------------------------
+
+- Inferred defaults used to create connection if none configured explicitly:
+
+  - credentials (derived from GAE / GCE environ if present).
+
+  - ``project_id`` (derived from GAE / GCE environ if present).
+
+  - ``scopes``
+
+
+Manage topics for a project
+---------------------------
+
+Create a new topic for the default project:
+
+.. doctest::
+
+   >>> from gcloud.pubsub.topic import Topic
+   >>> topic = Topic('topic_name')
+   >>> topic.create()  # API request
```

Comment:
I don't know that these docs actually stick around, but I wanted to make it clear where in the flow an actual back-end request would be made (being explicit, rather than the DWIM pattern we fell into originally with storage).